init/tmpfiles.py

```diff
@@ -46,6 +50,70 @@ def list_tmpfiles_configs():
     )
 
 
+def get_specifier_map():
+    """Create mapping from specifier to value"""
+    # $TMPDIR, $TEMP, $TMP, or /tmp
+    tmpdir = tempfile.gettempdir()
+
+    uname = os.uname()
+    # machine is "x86_64", systemd wants "x86_64"
+    arch = uname.machine.replace("_", "-")
+    hostname = uname.nodename
+    shortname = hostname.split(".", 1)[0]
+
+    with open(BOOT_ID) as f:
+        # Kernel boot_id file has dashes, systemd strips dashes
+        boot_id = f.read().strip().replace("-", "")
+
+    with open(MACHINE_ID) as f:
+        machine_id = f.read().strip()
+
+    return {
+        "a": arch,  # architecture
+        # "A": None,  # os-release IMAGE_VERSION or empty string
+        "b": boot_id,
+        # "B": None,  # os-release BUILD_ID or empty string
+        "C": "/var/cache",  # system cache dir
+        "g": "root",  # user group name
+        "G": "0",  # user gid
+        "h": "/root",  # home dir
+        "H": hostname,  # node host name
+        "l": shortname,  # short host name
+        "L": "/var/log",  # system log dir
+        "m": machine_id,  # /etc/machine-id
+        # "M": None,  # os-release IMAGE_ID or empty string
+        # "o": None,  # os-release ID, never empty string (!)
+        "S": "/var/lib",  # system state dir
+        "t": "/run",  # system runtime dir
+        "T": tmpdir,  # system tmp dir
+        "u": "root",  # user name
```

Comment:
Should we actually find out the username/uid running the process?  It might not be `root`.
Same for `g`, `G`, `h` and `U`.

I'm happy to leave it hardcoded but we should at least put a
comment that it diverges from `systemd-tmpfiles`.

Reply:
Makes sense, it was a simple fix.

Reply:
Thanks!